lib/arelastic/builders/queries.rb

```diff
@@ -13,6 +13,7 @@ class Queries < Struct.new :name
         geo_bounding_box:    Arelastic::Queries::GeoBoundingBox,
         geo_distance:        Arelastic::Queries::GeoDistance,
         geo_polygon:         Arelastic::Queries::GeoPolygon,
```

Comment:
This line is causing `lib/arelastic/queries/geo_polygon.rb` to be loaded, so the warning is always printed.  Can you wrap this somehow so that the class will only be auto-loaded if the `geo_polygon` method is called?